### ISSUE TITLE
Move whiltelabel options to DB and admin area

### ISSUE THREAD

**Issue by rene-mueller** [calcom/cal.com#10035]
### Is your proposal related to a problem?

Currently, all settings for branding should be moved to the admin area i.e., the following environment variables:

```
NEXT_PUBLIC_APP_NAME="Cal.com"
NEXT_PUBLIC_SUPPORT_MAIL_ADDRESS="help@cal.com"
NEXT_PUBLIC_COMPANY_NAME="Cal.com, Inc."
```

In addition, there could be added an upload for the logo?

**Comment by PeerRich** [comment]
yes agree. as much as possible should be moved into DB and configurable via Admin UI vs .env

ideally our .env is the bare minimum technically possible

@rene-mueller wanna work on this?

cc @leog

**Comment by rene-mueller** [comment]
@PeerRich that would be great

**Comment by danrfletcher** [comment]
<img width="376" alt="image" src="https://github.com/calcom/cal.com/assets/57096365/a45fe937-f1b1-45bf-9ad2-71bb47613751">

<img width="71" alt="image" src="https://github.com/calcom/cal.com/assets/57096365/67cdf5fb-3c5a-4c78-abeb-6946c3a07d9c">

Could this be an additional step in the license form/new form in admin area?
